UI/src/app/components/pages/mapping/mapping.component.scss

```diff
@@ -131,3 +131,39 @@ footer {
     color:$cdm-grey-dark;
   }
 }
+
+::ng-deep .mat-tab-labels {
```

Comment:
please get rid of ng-deep. It's deprecated in 10 angular. For future updates it'll cause double work.

To apply these styles  - they could be written in global scss files (preferable way to do this, e.g. create separate material styles file), or for now you can set  encapsulation: ViewEncapsulation.None

Reply:
P.S. this comment is about all ng deep in this PR

Reply:
I will fix it in next PR

Reply:
[Get rid of ng-deep](https://github.com/SoftwareCountry/CDMSouffleur/issues/57)